pandas_gbq/gbq.py

```diff
@@ -6,32 +6,20 @@
 from datetime import datetime
 import logging
 import re
-import time
-import typing
-from typing import Any, Dict, Optional, Sequence, Union
 import warnings
 
-import numpy as np
-
-# Only import at module-level at type checking time to avoid circular
-# dependencies in the pandas package, which has an optional dependency on
-# pandas-gbq.
-if typing.TYPE_CHECKING:  # pragma: NO COVER
-    import pandas
-
-import pandas_gbq.constants
-import pandas_gbq.exceptions
-from pandas_gbq.exceptions import GenericGBQException, QueryTimeout
+from pandas_gbq.contexts import context
+from pandas_gbq.exceptions import (
```

Comment:
Let's be explicit about what aliases we are keeping around with `__all__` + a comment explaining why these "unused" aliases are here.

Reply:
`__all__` may give us extra burden due to potential star imports. I used `# noqa` to mark all unused imports instead.